shuup/core/models/_product_shops.py

```diff
@@ -42,6 +42,7 @@ class ShopProduct(MoneyPropped, models.Model):
     visibility_groups = models.ManyToManyField(
         "ContactGroup", related_name='visible_products', verbose_name=_('visible for groups'), blank=True
     )
+    orderable_minimum = QuantityField(default=0, verbose_name=_('orderable minimum'))
```

Comment:
Could this be blank=True, null=Truem where the blank value would indicate the endles amount?